service/templates/admin.html

```diff
@@ -1745,18 +1745,6 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>Task Output S3 Secret</label>
 										<input type="text" ng-model="Settings.providers.aws.task_output.secret">
 									</md-input-container>
-									<md-input-container class="control" style="width:45%;">
```

Comment:
This particular table is super annoying because it has the width property alternate. Every time I add/remove an odd number of things I have the adjust all the following ones.

Reply:
I fixed the pattern, so it should look like it did before.